src/components/AuthButtons.tsx

```diff
@@ -0,0 +1,45 @@
+import { Box, Button } from "@mui/material";
+import { useNavigate } from "react-router-dom";
+
+type AuthButtonsProps = {
+  flexDirection?: "row" | "column";
+  hidden?: boolean;
+};
+
+const AuthButtons = ({ flexDirection = "row", hidden }: AuthButtonsProps) => {
+  const navigate = useNavigate();
+  return (
+    <Box
+      sx={{
+        display: { xs: hidden ? "none" : "flex", md: "flex" },
+        flexDirection: flexDirection,
+        gap: 2,
+      }}
+    >
+      <Button
+        onClick={() => navigate("/login")}
```

Comment:
Create RoutePages.ts file in your routes folder.
In that file, create an enum RoutePages {
Login = '/login',
Recipes = '/recipes' //this is just an example
}

Use the RoutePages enum for handling routes related logic. E.g.:

    onClick={() => navigate(RoutePages.Login)}

Route names tend to change, it's easier to maintain them from one place.